src/server/oauth/controllers/oauth.controller.js

```diff
@@ -1,42 +1,53 @@
+const { ServerError } = require('../../utils/error.utils');
```

Comment:
This was deleted in my PR, should use this instead:

`const errors = require('../../utils/error.utils');`